server/mqtt.go

```diff
@@ -270,6 +274,7 @@ type mqttRetMsgDel struct {
 
 type mqttSession struct {
 	mu                     sync.Mutex
+	subsMu                 sync.RWMutex
```

Comment:
NS: 1/5 consider making this a sync.WaitGroup as well. Need to research vs RWMutex